stdlib/2and3/numbers.pyi

```diff
@@ -22,42 +24,42 @@ class Complex(Number):
         def __nonzero__(self) -> bool: ...
     @property
     @abstractmethod
-    def real(self): ...
+    def real(self: _T) -> _T: ...
     @property
     @abstractmethod
-    def imag(self): ...
+    def imag(self: _T) -> _T: ...
     @abstractmethod
-    def __add__(self, other): ...
+    def __add__(self: _T, other: Any) -> _T: ...
     @abstractmethod
-    def __radd__(self, other): ...
+    def __radd__(self: _T, other: Any) -> _T: ...
     @abstractmethod
-    def __neg__(self): ...
+    def __neg__(self: _T) -> _T: ...
     @abstractmethod
-    def __pos__(self): ...
-    def __sub__(self, other): ...
-    def __rsub__(self, other): ...
+    def __pos__(self: _T) -> _T: ...
+    def __sub__(self: _T, other: Any) -> _T: ...
+    def __rsub__(self: _T, other: Any) -> _T: ...
     @abstractmethod
-    def __mul__(self, other): ...
+    def __mul__(self: _T, other: Any) -> _T: ...
     @abstractmethod
-    def __rmul__(self, other): ...
+    def __rmul__(self: _T, other: Any) -> _T: ...
     if sys.version_info < (3, 0):
         @abstractmethod
         def __div__(self, other): ...
         @abstractmethod
         def __rdiv__(self, other): ...
     @abstractmethod
-    def __truediv__(self, other): ...
+    def __truediv__(self: _T, other: Any) -> _T: ...
     @abstractmethod
-    def __rtruediv__(self, other): ...
+    def __rtruediv__(self: _T, other: Any) -> _T: ...
     @abstractmethod
-    def __pow__(self, exponent): ...
+    def __pow__(self: _T, exponent: Any) -> _T: ...
     @abstractmethod
-    def __rpow__(self, base): ...
-    def __abs__(self): ...
-    def conjugate(self): ...
-    def __eq__(self, other: object) -> bool: ...
+    def __rpow__(self: _T, base: Any) -> _T: ...
+    def __abs__(self: _T) -> _T: ...
```

Comment:
The absolute value of a complex number isn't a complex, it's a real. For builtins.complex it returns a float.